src/jit/lir.cpp

```diff
@@ -1040,8 +1168,14 @@ LIR::Range LIR::AsRange(GenTree* firstNode, GenTree* lastNode)
 //
 // Return Value: The newly constructed range.
 //
-LIR::Range LIR::SetTreeSeq(Compiler* compiler, GenTree* tree)
+// static
+LIR::Range LIR::SeqTree(Compiler* compiler, GenTree* tree)
 {
+    compiler->gtSetEvalOrder(tree);
```

Comment:
I am actually a bit leery of folding the `gtSetEvalOrder` call into this method, as it is not always necessary. It is probably never wrong to make this call, however, so leaving it here should be okay.